landing_page/app/helpers/sessions_helper.rb

```diff
@@ -1,23 +1,20 @@
 module SessionsHelper
+  # Logs the user in
```

Comment:
Is this comment really necessary? Is the method below readable?

Reply:
The method below is readable so I will remove this line

Reply:
Resolved in commit 468cf39e6692825efc60b3b265ef0896ab6f1b4a